### ISSUE TITLE
Shift not overruled by larger breakpoint?

### ISSUE THREAD

**Issue by 0x80** [garetmckinley/hedron#67]
In the flowing code, the shift is not 0 at md and above. Am I misunderstanding the API? I would expect the shift only to occur from sm to md.

```javascript
<Row divisions={9} debug>
  <Column md={3}>column 1</Column>
  <Column md={3}>column 2</Column>
  <Column md={3} mdShift={0} smShift={1}>column 3</Column>
</Row>
```

**Comment by garetmckinley** [comment]
Thanks for the issue report! This is actually a bug, `mdShift` is not overriding `smShift` because the value for `mdShift` is `0`, which is equal to no value. Since hedron thinks there's no value set for `mdShift`, it doesn't override `smShift`. A workaround for now would be to use a string value for `mdShift`.

Like this: https://www.webpackbin.com/bins/-KhYZwKLLE0C5JuNjHik

It's going to spit out a warning when you're in development mode (since it's expecting a number, not a string). But in production mode there will be no error (since `PropTypes` aren't used in production).

This issue is not present in hedron >`1.0.0` (installable via `hedron@next`), since the `shift` properties can accept strings, numbers, and objects.

**Comment by 0x80** [comment]
I was just playing around but it's good to know that the behavior I was expecting was correct. I will switch to 1.0 now, since that one seems to have a lot of improvements in general 👍

**Comment by garetmckinley** [comment]
It's not fully documented yet, but you can view the 1.0.0 storybook [here](https://35aca43a-4bb2-430d-afe9-a23e2a8d43cf.sbook.io/). It's still under heavy development, so if you have any questions or comments for us don't hesitate to reach out either on here or in our [slack group](https://slackin-xtuseyimsc.now.sh/)

**Comment by garetmckinley** [comment]
This was resolved via 1.0.0
